javascript/schema/src/validator.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+
+import type { Maybe } from '@data-wrangling-components/core'
+import Ajv from 'ajv'
+
+import type { WorfklowJson } from './schema.js'
+
+async function getSchema() {
+	/* eslint-disable @essex/extensions */
+	return import('./workflow.json').then(res => (res as any)?.default ?? {})
+}
+
+export async function schemaValidator(
```

Comment:
should this be `isSchemaValid`? schemaValidator implies that we're returning an AJV instance. Also, the Ajv instance should have a caching mechanism